example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -16,6 +16,7 @@
 		97C146FC1CF9000F007C117D /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 97C146FA1CF9000F007C117D /* Main.storyboard */; };
 		97C146FE1CF9000F007C117D /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 97C146FD1CF9000F007C117D /* Assets.xcassets */; };
 		97C147011CF9000F007C117D /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 97C146FF1CF9000F007C117D /* LaunchScreen.storyboard */; };
+		E5BD63083B32E98691A5A0B7 /* libPods-Runner.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 5E05894BC33939263F442533 /* libPods-Runner.a */; };
```

Comment:
Please run `pod deintegrate` command to keep the project clean and avoid potential conflicts in Xcode.